### ISSUE TITLE
interesting countered with nan loss too

### ISSUE THREAD

**Issue by zhhtu** [leeyoshinari/YOLO_v2#26]
It seems training loss encountered with NAN, then loss > 1e04 script stopped with "successful training". I've also trained yolo with MobileNet V1 also encountered with same NAN question. Maybe problem of feature extraction structure itself? Anyone tried successfully training with converge output?

**Comment by guker** [comment]
About Nan,  you can set batch_size=64, and use SGD optimizer
